tests/test_datasources.py

```diff
@@ -25,6 +25,7 @@
 import pytest
 
 from ansys import dpf
+from ansys.dpf.core import examples
 import conftest
 
 skip_always = pytest.mark.skipif(True, reason="Investigate why this is failing")
```

Comment:
@rafacanton @cbellot000 It looks like this test module is skipped for some reason